templates/404.html

```diff
@@ -2,18 +2,20 @@
 
 {% block title %}404: Page not found{% endblock %}
 
-{% block meta_description %}Sorry, we can't find the page you're looking for.{% endblock %}
+{% block meta_description %}We can't find the page you're looking for.{% endblock %}
 
 {% block content %}
-<div class="p-strip is-deep u-extra-space">
-  <div class="row">
-    <div class="col-6 col-medium-3 u-hide--small">
-      <img src="https://assets.ubuntu.com/v1/4de1c56c-404_v1.svg" alt="Error owl" width="400" height="400" />
+<div class="p-strip">
+  <hr class="p-rule is-fixed-width">
+  <div class="row p-section">
```

Comment:
Should we change this to a deep strip as well? Again, from Vanilla: "Sections should not be nested inside each other or inside the strips."

Reply:
yes thanks